11/main.js

```diff
@@ -0,0 +1,294 @@
+    productCountry: "USA",
+    productPrice: 16,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: false,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 145,
+  },
+  {
+    productName: "Jagermeister",
+    productData: {
+      weight: 56,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "UK",
+    productPrice: 115,
+  },
+  {
+    productName: "Spaghetti",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "Italia",
+    productPrice: 16,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 211,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 211,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 211,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 211,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 211,
+  },
+  
+];
+let i = 0;
+const div = document.createElement("div");
+div.className = "box";
+document.body.append(div);
+function createListAndInsertToHTML() {
+  const ul = document.createElement("ul");
+  div.append(ul);
+  for (let key of listOfProducts) {
+    const li = document.createElement("li");
+    li.innerHTML = key.productName;
+    i--;
+    ul.append(li);
+    const childUl = document.createElement("ul");
+    li.append(childUl);
+    for (let parameter in key) {
+      i++;
+      if (parameter !== "productName" && !(key[parameter] instanceof Object)) {
+        const childLi = document.createElement("li");
+        childLi.innerHTML = `${parameter} : ${key[parameter]}`;
+        isElementTwin(childLi, i);
+        childUl.append(childLi);
+        if (parameter == "productCountry") {
```

Comment:
Strict equality here and below
```suggestion
        if (parameter === "productCountry") {
```